tests/app/main/test_form_validators.py

```diff
@@ -0,0 +1,29 @@
+import mock
+import pytest
+
+from wtforms.validators import StopValidation
+
+from app.main.forms import AdminEmailAddressValidator, InviteAdminForm
+
+
+@mock.patch('app.main.forms.data_api_client')
+class TestAdminEmailAddressValidator(object):
+
+    def setup_method(self):
+        self.form_mock = mock.MagicMock(InviteAdminForm)
+        self.form_mock.email_address.data = 'the_email_address'
+
+    def test_admin_email_address_validator_calls_api(self, data_api_client):
+        AdminEmailAddressValidator()(self.form_mock, self.form_mock.email_address)
+        assert data_api_client.email_is_valid_for_admin_user.called_once_with('the_email_address')
+
+    def test_admin_email_address_validator_raises_with_invalid_response(self, data_api_client):
+        data_api_client.email_is_valid_for_admin_user.return_value = False
+
+        with pytest.raises(StopValidation, message='The email address must belong to an approved domain'):
```

Comment:
This test class is for the validator - should it know about the `InviteAdminForm`'s message? Can we test what happens when the default message (`None`, I think) is used?

Reply:
So the None case is just what happens with all validators (raises with no message) but I've changed these tests to agnostic of the invite form now